src/sg/gov/tech/geekout/1.0/geekout-open-attestation.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$id": "https://schemata.openattestation.com/sg/gov/tech/geekout/1.0/geekout-open-attestation",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://schema.openattestation.com/2.0/schema.json"
+    },
+    {
+      "$ref": "schema.json"
```

Comment:
actually I'm not sure this will behave correctly, let's use the full url to the geekout schema